plugins/rapid7_insightvm/komand_rapid7_insightvm/util/update_shared_credential_util.py

```diff
@@ -0,0 +1,207 @@
+from typing import Union
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+
+from komand_rapid7_insightvm.util.util import check_in_enum, check_not_null
+
+
+def ssh_setup(account: dict) -> tuple[str, str, str]:
```

Comment:
I am up for using the `Dict` from typing library 😄 Add there necessary structure types as well for example `Dict[str, Any]`
```suggestion
def ssh_setup(account: Dict[str, Any]) -> tuple[str, str, str]:
```